<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QcdmWindow</class>
 <widget class="QMainWindow" name="QcdmWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QCDM - Service Programming</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="qcdmTabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>782</width>
      <height>231</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabSecurity">
     <attribute name="title">
      <string>General</string>
     </attribute>
     <widget class="QPushButton" name="readMeidButton">
      <property name="geometry">
       <rect>
        <x>151</x>
        <y>9</y>
        <width>51</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Read</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>14</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>MEID</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="meidValue">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>14</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>44</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>IMEI</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="imeiValue">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>15</number>
      </property>
     </widget>
     <widget class="QPushButton" name="readImeiButton">
      <property name="geometry">
       <rect>
        <x>151</x>
        <y>39</y>
        <width>51</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Read</string>
      </property>
     </widget>
     <widget class="QPushButton" name="readSpcButton">
      <property name="geometry">
       <rect>
        <x>91</x>
        <y>169</y>
        <width>51</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Read</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>174</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> SPC</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="securitySpcValue_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>51</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>6</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabNvEditor">
     <attribute name="title">
      <string>NAM</string>
     </attribute>
     <widget class="QComboBox" name="nvItemList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>321</width>
        <height>27</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="log">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>781</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="portList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="portListRefreshButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>9</y>
      <width>61</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="portDisconnectButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>9</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="portConnectButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>9</y>
      <width>61</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="programNameText">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OpenPST</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Security</string>
    </property>
    <widget class="QPushButton" name="securitySendSpcButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>19</y>
       <width>51</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>159</x>
       <y>24</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="indent">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="security16PasswordValue">
     <property name="geometry">
      <rect>
       <x>209</x>
       <y>20</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>FFFFFFFFFFFFFFFF</string>
     </property>
     <property name="maxLength">
      <number>16</number>
     </property>
    </widget>
    <widget class="QPushButton" name="securitySend16PasswordButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>19</y>
       <width>51</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>24</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string> SPC</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="securitySpcValue">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>20</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>000000</string>
     </property>
     <property name="maxLength">
      <number>6</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Extra</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
